### DISCUSSION TITLE
Output colored text from file to #draw box

### DISCUSSION THREAD

**Discussion by thechanster** [scandum/tintin#189] [Script help]
Hey, i've written output to a file, including color codes. Looking to import from the file to a #draw box, using:

    #scan {file} {./outputs/recentchat.txt} {#var recentchat &0};
    #draw white unicode box $commsboxa $commsboxb $commsboxc $commsboxd $recentchat;

However, seems the #scan command tries to execute the output, rather than just store to the variable, causing a bunch of unknown command erros from the MUD.

Am I reading the file into a variable in in the right way?  Is there a better way. Want to maintain the color codes.

Thanks in advance! Matt.

**Comment by scandum** [comment]
There's probably a `;` in the file, so you'll need to use `{&0}`
```
#scan {file} {./outputs/recentchat.txt} {#var recentchat {&0}};
```
